include/oneapi/dpl/pstl/hetero/algorithm_ranges_impl_hetero.h

```diff
@@ -129,6 +174,20 @@ __pattern_find_if(__hetero_tag<_BackendTag>, _ExecutionPolicy&& __exec, _Range&&
         _Predicate{__pred}, _TagType{}, ::std::forward<_Range>(__rng));
 }
 
+#if _ONEDPL___cplusplus >= 202002L
```

Comment:
Should it rather rely on `_ONEDPL_CPP20_RANGES_PRESENT`?

Reply:
sure, good catch!